code/datums/components/crafting/crafting.dm

```diff
@@ -184,6 +184,9 @@
 				I = new R.result (get_turf(a.loc), R.result_amount || 1)
 			else
 				I = new R.result (get_turf(a.loc))
+				if(istype(I, /obj/item/storage))
```

Comment:
I feel like there's a better way to do this than to spawn the contents and then qdel them.
Maybe a way to prevent PopulateContents from being called

Reply:
> Check for `atom_storage` rather than isyping the storage subtype?
> 
> ~Unless there are some non-storage crafting recipes that are supposed to start full~

Hm, I can't think of any right now soooooooooooo...

Reply:
@ZephyrTFA In a perfect world, I think `mapload: bool` would instead be `creation_source: "mapload" | "craft" | etc`, and we would check on that, but with what we have now I'm happy enough with this